File: Backend/Models/AppUser.cs
using Microsoft.AspNetCore.Identity;

namespace Backend.Models
{
    public class AppUser : IdentityUser
    {
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public List<MailBox> MailBoxes { get; set; } = [];
    }
}

File: frontend/src/models/appUser.ts


export interface AppUser {
  id: string,
  email: string,
  emailConfirmed: boolean,
  twoFactorEnabled: boolean,
  firstName?: string,
  lastName?: string
}

export interface EditAppUser{
  id: string,
  firstName: string|null,
  lastName: string|null
}